test_bot/test_bot.py

```diff
@@ -72,18 +72,23 @@ def download_lc0() -> None:
         zip_ref.extractall("./TEMP/")
 
 
-def download_sjeng() -> None:
-    """Download Sjeng."""
-    if os.path.exists("./TEMP/sjeng.exe"):
+def download_arasan() -> None:
+    """Download Arasan."""
+    if os.path.exists(f"./TEMP/arasan{file_extension}"):
         return
-
-    response = requests.get("https://sjeng.org/ftp/Sjeng112.zip", allow_redirects=True)
-    response.raise_for_status()
-    with open("./TEMP/sjeng_zip.zip", "wb") as file:
+    if platform == "win32":
+        response = requests.get("https://arasanchess.org/arasan24.2.2.zip", allow_redirects=True)
+    else:
+        response = requests.get("https://arasanchess.org/arasan-linux-binaries-24.2.2.tar.gz", allow_redirects=True)
+    with open(f"./TEMP/arasan.{archive_ext}", "wb") as file:
```

Comment:
You need `response.raise_for_status()` before this line so a failed download raises an exception.